.travis.yml

```
@@ -2,7 +2,11 @@ sudo: required
 language: python
 python:
   - "3.6.4"
-# command to run tests
+services: 
+    - docker
+before_install: 
+    - docker build -t jr/easycrm -f Dockerfile.dev .
```

```suggestion
services: 
  - docker
before_install: 
  - docker build -t jr/easycrm -f Dockerfile.dev .
```
Prefer to keep consistency format (1 tab or 2 spaces) here.

Also we are missing the `Dockerfile.dev`, maybe we should use `Dockerfile` instead